tutorials/casper/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Casper 
+
+```bash
+module purge
+module load gnu/9.1.0 ncarcompilers openmpi netcdf ncarenv cmake
+```
```

Comment:
might want to put a note in that this does NOT work with gcc 10 (for future reference)

Reply:
It does work with gnu 10 and gnu 11.  I've removed the version number.

Reply:
SmartRedis, possibly, SmartSim backends will not.

see https://github.com/RedisAI/RedisAI/issues/777 

Confirmed by RedisAI devs as well.